android/lib/AndroidPlatform.js

```diff
@@ -1389,21 +1392,6 @@ function(configId, args, callback) {
         return;
     }
 
-    // Extract lzma libxwalkcore, because we are not supporting
```

Comment:
We have java class which can decompress in runtime. In Lite, the .so file is compressed by default in xwalk_core_library, and it is decompressed in runtime.
So in app-tools, we don't need the decompress function.